docs/modules/ROOT/pages/advanced-how-tos/using-trusted-artifacts.adoc

```diff
@@ -0,0 +1,38 @@
+= Using Trusted Artifacts
+
+Konflux relies on Trusted Artifacts to securely allow sharing of data between Tasks without
+compromising the build process. See
+link:https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html[ADR-36] for more
+information.
```

Comment:
This statement makes it sound like the use of trusted artifacts is universal. Should we instead indicate that it is one of the operation modes?

Reply:
Is it not the intended universal operation mode?

Reply:
While PVCs are more limited in terms of their customization, they will likely be more efficient and faster if builds involve many GBs of data being transmitted between tasks.

There will be a heavy preference for it to be the default but that doesn't _necessarily_ mean that it will be the only option.

Reply:
Fair point. Rewrote this paragraph to mention the two modes.